package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ipc144",
   "version": "0.0.0",
   "private": true,
-  "license" : "GFDL-1.3-no-invariants-or-later",
```

Comment:
Please revert changes to this file, it's not needed here.

Reply:
Done with it, could you please check if you like it?